### DISCUSSION TITLE
Allow unsafe enchantments in anvils

### DISCUSSION THREAD

**Discussion by elkoyofficial** [PurpurMC/Purpur#963] [__Completed__] [answered]
Add the option to enable unsafe enchantments in anvils. Currently, if you combine two weapons with one having unsafe enchantments, it will only transfer the max vanilla enchant level.

https://gyazo.com/a68f140d56ef18ef0946f7cea444d900

**Comment by RhythmicSys** [comment, marked as answer]
This works in the current builds :3
https://user-images.githubusercontent.com/45906780/184510231-e570a5b9-c4cc-49fe-bec4-32a0d47d68ae.mp4

```    
anvil:
      allow-unsafe-enchants: true
      allow-inapplicable-enchants: true
      allow-incompatible-enchants: true
      allow-higher-enchants-levels: true
    allow-unsafe-enchant-command: true
    clamp-levels: false```
